src/test/java/com/crud/tasks/config/TrelloConfigTestSuite.java

```diff
@@ -0,0 +1,52 @@
+package com.crud.tasks.config;
+
+import com.crud.tasks.trello.config.TrelloConfig;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.mockito.Mockito.when;
+
+@SpringBootTest
+@RunWith(MockitoJUnitRunner.class)
+public class TrelloConfigTestSuite {
+
+    @Mock
+    TrelloConfig trelloConfig;
+
+    @Test
+    public void testTrelloApiEndPoint() {
+        //Given
+        String apiEndPoint = "http://api.trello.com/1";
+        when(trelloConfig.getTrelloApiEndpoint()).thenReturn(apiEndPoint);
+        //When
+        String expectedApiEndPoint = trelloConfig.getTrelloApiEndpoint();
```

Comment:
1. Ten test w ogóle nie sprawdza `TrelloConfig`. Sprawdza, że mock działa tak jak go skonfigurowałeś. Testujesz mockito, nie klasę `TrelloConfig`.
2. To, co powstaje w `when`, jest przedmiotem testu. To, co jest założeniem, powstało w sekcji `given`. Zmienna tworzona w linii 27 nie powinna nazywać się `expected...`, - expected jest to, co stworzyłeś w linii 24.
3. Ogólnie test takiej klasy ma bardzo niewielką wartość. Czego się z niego dowiadujemy? Jak zabezpiecza nas przed błędami? To trochę test na sztuczne nabicie coverage (a i tak tego nie zrobi, bo testuje mocka - spróbuj odpalić coverage ale tylko z tym jednym testem, nie z wszystkimi na raz).